work2/myworld/members/views.py

```diff
@@ -0,0 +1,43 @@
+from django.views import View
+from .models import Employees
+from django.http import JsonResponse
+from django.views.decorators.csrf import csrf_exempt
+from django.utils.decorators import method_decorator
+import pdb
+
+@method_decorator(csrf_exempt, name='dispatch')
+class EmployeeView(View):
+    def get(self, request, empid=None, department=None):
+        pdb.set_trace()
+        employee_model_list = []
+        try:
+            if empid:
+                employee_model_list = Employees.objects.filter(emp_id=empid)
+            elif department:
+                employee_model_list = Employees.objects.filter(department=department)
+        except Employees.DoesNotExist:
+            return JsonResponse({'status': 'failed', "employees": None}, status=400)
+        employees = []
+        for employee in employee_model_list:
+            data = {
+                    "first_name" : employee.first_name,
+                    "last_name": employee.last_name,
+                    "address": employee.address,
+                    "roll_number": employee.emp_id,
```

Comment:
@sampleharvgithub Shouldn't this be the employee id?

Reply:
I kept copy of the folder and made changes ..and i have uploaded the wrong one.. now i have committed the changes